### DISCUSSION TITLE
Customizing the navigation and card

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5696] [Q&A]
Customizing the navigation and card[#](https://statamic.dev/extending/utilities#customizing-the-navigation-and-card)
You can customize the nav item and the card within the listing by chaining additional methods.

**$utility
    ->title('French Fries')
    ->navTitle('Fries')
    ->description('Makes french fries at the click of a button.')
    ->docsUrl('https://yoursite.com/docs/french-fries')
    ->register();**

Where to put this ?

**Comment by jasonvarga** [comment]
In a service provider's boot method.

![image](https://user-images.githubusercontent.com/105211/160855873-da48f1bd-02f4-478b-9280-d0ec02e16cae.png)
